### PR TITLE
Remove unused related resources json files

### PR DESCRIPTION
**Pull request by LizBaker**
We removed the related resources script but still have the data files from it. This is some clean up 🧹

### CODE REVIEW

File: src/templates/docPage.js [hunk 1]
```diff
@@ -109,12 +107,6 @@ const BasicDoc = ({ data, location, pageContext }) => {
             />
           )}
           <TableOfContents headings={headings} />
-          <RelatedResources
```
Comment (roadlittledawn) [inline src/templates/docPage.js:112]: what happens if a content author adds related resources via frontmatter?

Comment (LizBaker) [reply src/templates/docPage.js]: they become very saddened when it doesn't show up... 😂 just reverted that

**Review by roadlittledawn — APPROVED** [review]
thanks for the catch and clean up!

**Comment by LizBaker** [comment]
close/open to retrigger build

**Review by roadlittledawn — DISMISSED** [review]
:shipit:

**Review by roadlittledawn — APPROVED** [review]

**Pull request merged**